apps/aehttp/priv/swagger.json

```diff
@@ -532,6 +551,12 @@
             "schema" : {
               "$ref" : "#/definitions/Info"
             }
+          },
+          "401" : {
```

Comment:
I don't know if 401 (Unauthorized) is requirement, but I think 403 (Forbidden) if semantically better.

https://httpstatuses.com/401
https://httpstatuses.com/403